### PR TITLE
neutron: Add ability to configure the f5_ha_type option for F5 LBaaS + one fix

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: crowbar_framework/app/helpers/barclamp/neutron_helper.rb [hunk 1]
```diff
@@ -73,6 +73,17 @@ def networking_lbaasv2_drivers(selected)
       )
     end
 
+    def f5_ha_type(selected)
+      options_for_select(
+        [
+          [t(".f5.ha_types.standalone"), "standalone"],
+          [t(".f5.ha_types.pair"), "pair"],
+          [t(".f5.ha_types.scalen"), "scalen"]
```
Comment (matelakat) [inline crowbar_framework/app/helpers/barclamp/neutron_helper.rb]: Might be a stupid question: What does `".f5.ha_types.scalen"` reference? Is it just an arbitrary string?

Comment (vuntz) [inline crowbar_framework/app/helpers/barclamp/neutron_helper.rb]: This string: https://github.com/crowbar/crowbar-openstack/pull/788/files#diff-1671f88e92d191883e65b12ee99abf44R72

Comment (matelakat) [inline crowbar_framework/app/helpers/barclamp/neutron_helper.rb]: thanks!

**Comment by vuntz** [comment]
@matelakat @aplanas @aspiers are you still +1? I had force-pushed to add the missing migration (that was not there due to me forgetting to do "git add").

**Pull request merged**